### PR TITLE
Update troubleshooting.md to include REFRESH OAUTH TOKEN

### PR DESCRIPTION
**Pull request by rashamalek**
Update troubleshooting.md to include REFRESH OAUTH TOKEN button

### CODE REVIEW

**Comment by rashamalek** [comment]
:+1:

**Comment by scherniavsky** [comment]
:+1:

**Comment by scherniavsky** [comment]
:+1:

**Comment by scherniavsky** [comment]
:+1:

**Comment by scherniavsky** [comment]
:+1:

**Comment by scherniavsky** [comment]
:+1:

**Comment by scherniavsky** [comment]
:+1:

**Comment by fokusferit** [comment]
👍

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -50,13 +50,13 @@ Until now there were two reasons why this happened:
...
  
-**Solution**: Disable and enable the Zappr check to replace the stored token.
+**Solution**: Use REFRESH OAUTH TOKEN button to refresh the stored token.
 
 ### API errors
 
 Services Zappr depends on return http status code > 499.
 
-**Solution**: Disable and enable the Zappr check to retry the services call.
+**Solution**: Use REFRESH OAUTH TOKEN button to refresh the stored token and retry the services call.
```
Comment (harti2006) [inline docs/troubleshooting.md]: is there really a retry when you refresh the token? I'd rather guess you have to close and reopen the PR to retry.

Comment (rashamalek) [reply docs/troubleshooting.md]: actually there is, no need to open and close the pr afaik

**Comment by scherniavsky** [comment]
:+1:

File: docs/troubleshooting.md (hunk 1 above)
Comment (fokusferit) [reply docs/troubleshooting.md]: what does retry services call mean? Refreshing page ?

**Comment by rashamalek** [comment]
:+1:

**Comment by scherniavsky** [comment]
:+1:

**Comment by rashamalek** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**